CMakeLists.txt

```diff
@@ -40,19 +39,21 @@ if(MSVC)
 endif()
 
 if(WITH_OTLP)
-  set(WITH_PROTOBUF ON)
-endif()
-
-if(WITH_PROTOBUF)
   set(protobuf_MODULE_COMPATIBLE ON)
-  find_package(Protobuf)
+  find_package(protobuf)
```

Comment:
I think latest actually requires `Protobuf` with capital P?

Reply:
Restored the casing for now. Both should work, `protobuf` is probably better as `protobuf-config.cmake` comes from protobuf project while `FindProtobuf.cmake` is provided as built-in module by cmake.

Reply:
My comment based on what's written in CMake built-in, and I hit this issue with casing elsewhere on older Ubuntu-14.xx .. thus, the caution. Maybe it doesn't matter.